### PR TITLE
Fix SGB output lines documentation

### LINKED ISSUE #546: SGB command packet transfers are described using wrong bits of JOYP
On the page for SGB Command Packet Transfers, they are consistently described as using bits 6 and 7 of the JOYP register, when in reality it should be bits 4 and 5. This is described correctly [on the old version of the docs](https://gbdev.gg8.se/wiki/articles/SGB_Functions#SGB_Command_Packet_Transfers):

> Command packets (aka Register Files) are transferred from the gameboy to the SNES by using P14 and P15 output lines of the JOYPAD register (FF00h), these lines are normally used to select the two rows in the gameboy keyboard matrix (which still works).

Proof that this is indeed how the feature works can be found in the code for games such as [Pokemon Gold](https://github.com/pret/pokegold/blob/master/engine/gfx/color.asm#L712), where the values $10 (a 1 bit), $20 (a 0 bit), and $30 (no data) are used for the transfer.

### PR DESCRIPTION
**Pull request by Zarithya**
Fixes #546.

### CODE REVIEW

**Review by avivace — APPROVED** [review]

File: src/SGB_Command_Packet.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Command Packet Transfers
 
-Command packets are transferred from the Game Boy to the SNES by using bits 6 and 7 of of [the `JOYP` register][JOYP].
+Command packets are transferred from the Game Boy to the SNES by using bits 4 and 5 of of [the `JOYP` register][JOYP]. These lines are normally used to select the two rows in the Game Boy keyboard matrix (which still works).
```
Comment (ISSOtm) [inline src/SGB_Command_Packet.md:3]:
```suggestion
Command packets are transferred from the Game Boy to the SNES by using bits 4 and 5 of of [the `JOYP` register][JOYP].
These lines are normally used to select one of the two button groups (which [still works as usual][JOYP]).
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
I think this is importing the wording from the historical doc? I suggest rewording it a little so it's a bit clearer.

Thank you for this PR! ^^

**Comment by Zarithya** [comment]
@ISSOtm Wasn't sure what wording to use there so I erred on the side of using the same wording as the old doc. I've accepted your change into the PR.

**Comment by ISSOtm** [comment]
(GitHub tip: you can [dismiss a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review) and then re-request it!)

**Review by ISSOtm — APPROVED** [review]
Thanks!
